manager/controllers/utils.go

```diff
@@ -507,28 +507,14 @@ func systemDBLookups2SystemItems(systems []SystemDBLookup) ([]SystemItem, int, m
 			utils.LogDebug("err", err.Error(), "inventory_id", systems[i].ID, "system groups parsing failed")
 		}
 		data[i] = SystemItem{
-			Attributes: systems[i].SystemItemAttributesAll,
```

Comment:
shouldn't this stayed as `SystemItemAttributesAll`? We probably need to return more attributes similarly to SystemDetail handler